clio-server/src/test/scala/org/broadinstitute/clio/server/webservice/GvcfWebServiceSpec.scala

```diff
@@ -112,14 +113,12 @@ class GvcfWebServiceSpec extends BaseWebserviceSpec {
           // scolds us for using .head
           fail("Impossible because of the above check")
         }
-        .as[DocumentGvcf]
-        .fold(throw _, identity)
-
-      firstUpdate.upsertId should be(responseAs[UpsertId])
-      firstUpdate.gvcfMd5 should be(Some('abcgithashdef))
-      firstUpdate.notes should be(Some("some note"))
-      firstUpdate.gvcfPath should be(
-        Some(URI.create(s"gs://path/gvcf${GvcfExtensions.GvcfExtension}"))
+
+      ElasticsearchIndex.getUpsertId(firstUpdate) should be(responseAs[UpsertId])
+      getStringByName(firstUpdate, "gvcf_md5") should be("abcgithashdef")
```

Comment:
Theoretically this is a `Symbol`, but I couldn't get `ScalaTest` to believe me that it was a `Symbol`:
```
[info] - should upsert a record, delete it and then fail to find it with query, but find it with queryall *** FAILED ***
[info]   'abcgithashdef has neither an abcgithashdef nor an isAbcgithashdef method (GvcfWebServiceSpec.scala:118)
```

Reply:
Yeah, scalatest treats `Symbol` args to `be` weirdly. If you want it to be `Symbol` you could use `equal` instead of `be`, but I'm OK leaving it as a string here.